src/site/_includes/components/updates/updates.njk

```diff
@@ -5,6 +5,8 @@
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds-from-desktop">
           <h2 class="govuk-heading-l">Updates</h2>
+          <p class="govuk-body"><span class="govuk-!-font-weight-bold">20 October 2023:</span> The Defence Developer Services (D2S) will shortly commence work on its re-use strategy and the discovery and delivery of a re-use components service. As part of this, we will commence a Discovery activity for the Design System Toolkit. During the Discovery Phase, the MOD.UK Frontend and Design Library will be in maintenance mode with no further enhancements being made during this time. Only high priority issues such as security fixes or outages will be completed.</p>
```

Comment:
Nitpick: I think this `span` should be a [`time` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) because that's more semantic, although you're just following the pattern that was already in place so it's fine to leave it as-is. I guess there's also [compatibility considerations](https://caniuse.com/mdn-html_elements_time) with the `time` element.